src/sdk-reference/go/1/essentials/error-handling/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: sdk.html.hbs
+algolia: true
+title: Error Handling
+description: How to handle errors with the SDK
+order: 100
+---
+
+# Error Handling
+
+Each method returns at least one structure corresponding to a possible error.  
+When this structure is different from nil, it is because the method has failed.  
+
+The returned structure is an `error` that can be casted to a `KuzzleError` and it contains 3 properties:
+
+| Property   | Type    | Description                       |
+| ---------- | ------- | --------------------------------- |
+| `Message` | string | Message describing the error |
```

Comment:
I guess that this `Message` property has been added to be used by SWIG, but the standard `error` interface already has a property to access to the error message: `Error()`.
Since that table details only new properties added by the `KuzzleError` type, then that line should be removed entirely.